code/controllers/subsystem/server_maint.dm

```diff
@@ -55,5 +55,8 @@ SUBSYSTEM_DEF(server_maint)
 			co.ehjax_send(data = "roundrestart")
 		if(server)	//if you set a server location in config.txt, it sends you there instead of trying to reconnect to the same world address. -- NeoFite
 			C << link("byond://[server]")
+	if(SERVER_TOOLS_PRESENT)
+		SSblackbox.record_feedback("text", "server_tools", null, SERVER_TOOLS_VERSION)
+	SSblackbox.record_feedback("text", "server_tools_api", null, SERVER_TOOLS_API_VERSION)
```

Comment:
Shouldn't this be under `if(SERVER_TOOLS_PRESENT)` too?

Reply:
API is part of the code, so it's always present